tests/unit/evaluation/MeanSquared_unittest.cc

```diff
@@ -0,0 +1,66 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Yuhui Liu
+ *
+ */
+#include "sg_gtest_utilities.h"
+#include <cmath>
+#include <memory>
+#include <shogun/evaluation/MeanAbsoluteError.h>
+#include <shogun/evaluation/MeanSquaredError.h>
+#include <shogun/evaluation/MeanSquaredLogError.h>
+#include <shogun/labels/BinaryLabels.h>
+#include <shogun/labels/RegressionLabels.h>
+
+class MeanSquaredTest : public ::testing::Test

```

Comment:
could you update this to use the type-templated tests? see an example here:
https://github.com/shogun-toolbox/shogun/blob/develop/tests/unit/base/SGObjectAll_unittest.cc#L29

probably some of these errors will fail for some types, but at least the tests for different floating point types should be tested and it should work

Reply:
When I try to use the type-templated tests, I find the parameter of `MSE->evaluate`is `std:shared_ptr<Labels>`, so, could you tell me which parameter should be templated?